main/main.cpp

```
@@ -9,48 +9,68 @@
     #include "freertos/task.h"
-    #include "esp_chip_info.h"
-    #include "esp_flash.h"
-    #include "esp_system.h"    
+    #include "esp_system.h"
     #include "driver/i2c.h"
-    #include "../components/shtc1/shtc1.h"
+    #include "../components/shtc1/sensirion_i2c.h"
 }
 
+// Callback function to handle measurements
+void measurementHandler(int32_t temperature, int32_t humidity) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'measurementHandler' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)